Automattic/newspack-plugin
# feat(ia): display settings - part 2

#### Upon #3418 merge this PRs destination will change to `epic/ia`.

### All Submissions:


### Changes proposed in this Pull Request:

Adds the remaining 3 sections to Display Settings tab on Newspack > Settings page.

### How to test the changes in this Pull Request:

1. Checkout this branch and build assets i.e. `git checkout origin/feat/ia-settings-display-settings__part-2 && npm run build`
2. Navigate to _/wp-admin/admin.php?page=newspack-settings#/display-settings_
3. Scroll down to "Default Featured Image Position And Post Template" and test:
   * Updating settings, save and ensure changes persist between page refreshes and tab transitions.
![Screenshot 2024-09-23 at 09 20 37](https://github.com/user-attachments/assets/ad5ba36d-b987-42f6-966c-2b0772e16a9e)
4. Navigate to next section "Featured Image Position And Post Template For All Posts" and test:
   * Updating settings and click "Save", this will issue a prompt requiring confirmation to issue changes across ALL posts.
![Screenshot 2024-09-23 at 10 03 17](https://github.com/user-attachments/assets/d4796cff-e9c5-4fe3-80cd-f5b039ab6c5e)
![Screenshot 2024-09-23 at 09 20 54](https://github.com/user-attachments/assets/91cda5db-c39b-4ce0-8969-32fdd2588b1d)
   * Confirm after saving that `select` field(s) reset to "Select to change all posts".
5. Navigate to next section "Media Credit" and test:
   * Updating settings, save and ensure changes persist between page refreshes and tab transitions.
![Screenshot 2024-09-23 at 09 21 29](https://github.com/user-attachments/assets/d6c08819-0947-4a90-b666-1a3ea038ff96)

### Other information:
